### PR TITLE
Chore/unify eslint rules

### PR DESCRIPTION
**Pull request by dpinol**
### Description
All packages now use a single .eslintrc.js file (some do minor modifications). Errors reported by the new configuration have been fixed or suppressed.

### Context
We had a unified .eslintrc.js on the packages folder, but some packages were not using it. This was inconsistent and difficult to maintain

### CODE REVIEW

File: packages/botonic-nlu/src/file-utils.js [hunk 1]
```diff
@@ -73,7 +74,11 @@ export function loadConfigAndTrainingData(nluPath, languages) {
       const devIntents = { intentsDict: {}, intents: [] }
       const devEntities = { words: {}, tags: {}, tagList: [] }
       for (const [idx, file] of utterancesFiles.entries()) {
-        devIntents.intentsDict[idx] = getIntentName(file)
+        const filename = getIntentName(file)
```
Comment (dpinol) [inline packages/botonic-nlu/src/file-utils.js:77]: @marcrabat can you review this, please?

File: packages/botonic-react/src/webview.jsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ class App extends React.Component {
         )
       } catch (e) {}
       try {
-        parent.postMessage('botonicCloseWebview', '*')
+        await parent.postMessage('botonicCloseWebview', '*')
```
Comment (dpinol) [inline packages/botonic-react/src/webview.jsx:54]: @marcrabat @ericmarcos can you review this, please?

File: packages/botonic-react/src/webchat-app.jsx [hunk 3]
```diff
@@ -49,7 +49,7 @@ export class WebchatApp {
 
   async onUserInput({ user, input }) {
     this.onMessage && this.onMessage(this, { from: 'user', message: input })
-    this.hubtypeService.postMessage(user, input)
+    return this.hubtypeService.postMessage(user, input)
```
Comment (dpinol) [inline packages/botonic-react/src/webchat-app.jsx:52]: @marcrabat @ericmarcos can you review this, please?

File: packages/botonic-nlu/src/file-utils.js (hunk 1 above)
Comment (vanbasten17) [reply packages/botonic-nlu/src/file-utils.js:77]: yes, better this way, thanks 👍

File: packages/botonic-react/src/webchat-app.jsx (hunk 3 above)
Comment (vanbasten17) [reply packages/botonic-react/src/webchat-app.jsx:52]: looks good to me

**Pull request merged**